src/sentry/static/sentry/app/components/userMisery.tsx

```diff
@@ -4,39 +4,47 @@ import ScoreBar from 'app/components/scoreBar';
 import Tooltip from 'app/components/tooltip';
 import CHART_PALETTE from 'app/constants/chartPalette';
 import {tct} from 'app/locale';
+import {defined} from 'app/utils';
 
 type Props = {
   bars: number;
   barHeight: number;
-  miserableUsers: number;
-  totalUsers: number;
+  userMisery: number;
   miseryLimit: number;
+  totalUsers: number | undefined;
+  miserableUsers: number | undefined;
 };
 
 function UserMisery(props: Props) {
-  const {bars, barHeight, miserableUsers, miseryLimit, totalUsers} = props;
+  const {bars, barHeight, userMisery, miseryLimit, totalUsers, miserableUsers} = props;
+  // User Misery will always be > 0 because of the maximum a posteriori estimate
+  // and below 5% will always be an overestimation of the actual proportion
+  // of miserable to total unique users. We are going to visualize it as
+  // 0 User Misery while still preserving the actual value for sorting purposes.
+  const adjustedMisery = userMisery >= 0.05 ? userMisery : 0;
 
   const palette = new Array(bars).fill([CHART_PALETTE[0][0]]);
-  const rawScore = Math.floor(
-    (miserableUsers / Math.max(totalUsers, 1)) * palette.length
-  );
-
-  const adjustedScore = rawScore > 0 ? rawScore : miserableUsers > 0 ? 1 : 0;
-
-  const miseryPercentage = ((100 * miserableUsers) / Math.max(totalUsers, 1)).toFixed(2);
-
-  const title = tct(
-    '[affectedUsers] out of [totalUsers] ([miseryPercentage]%) unique users waited more than [duration]ms',
-    {
-      affectedUsers: miserableUsers,
-      totalUsers,
-      miseryPercentage,
+  const score = Math.round(adjustedMisery * palette.length);
+
+  let title: React.ReactNode;
+  if (defined(miserableUsers) && defined(totalUsers)) {
+    title = tct(
+      '[miserableUsers] out of [totalUsers] unique users waited more than [duration]ms',
+      {
+        miserableUsers,
+        totalUsers,
+        duration: 4 * miseryLimit,
+      }
+    );
+  } else {
+    title = tct('Probability that users waited more than [duration]ms is [userMisery]', {
```

Comment:
I wonder if Probability might be a bit misleading here, cause even with a transaction that i put a `sleep(1200)` in (so it has a 100% probability of misery) the user_misery will be high, but never 100%

Should we try to refer to this new misery as a "score" instead so that we're more ambiguous?